sass/_theme_variables.sass

```
@@ -51,6 +51,7 @@ $nav-link-color:                      $blue
 $nav-link-color-visited:              $purple
 $nav-link-color-hover:                lighten($nav-link-color, 6%) !default
 $nav-link-color-alt:                  hsl(33, 100%, 51%)
+$nav-caption:                         $nav-search-background-color
```

```suggestion
$nav-caption:                         $blue
```
Might prevent headers becoming unreadable if the search background color (the blue box behind search and logo) is ever changed, e.g. to black or gray